hasher-matcher-actioner/webapp/package.json

```diff
@@ -5,8 +5,12 @@
   "dependencies": {
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
-    "@testing-library/user-event": "^12.8.1",
+    "@testing-library/user-event": "^13.0.1",
     "bootstrap": "^4.6.0",
+    "eslint": "^7.22.0",
```

Comment:
nit: Is it customary for things to be dependencies AND devDependencies?

While it is unlikely someone does `npm install <hma>`, this seems weird.

Reply:
good catch, these don't need to be in dependencies; moved them to devDependencies.